### PR TITLE
Aws kinesis connector

### PR DESCRIPTION
**Pull request by girishg4t**
Added new AWS Kinesis connector, below are the steps it perform

1.  Connect to kinesis data stream
2.  Finds the shards 
3.  Check if any shards are added or removed
4.  Read data from each shard in parallel and call the http endpoint
5.  Depending on the response if success it send message to response stream else send error in error stream

### CODE REVIEW

File: aws-kinesis-http-connector/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- `CONTENT_TYPE`: Content type used while creating post request.
+
+#### Ways to connect to AWS
+- `AWS_REGION`: Region is mandatory for any aws connection.
+  
+1) Through AWS endpoint  
+- `AWS_ENDPOINT` : Kinesis endpoint on which it is running, for local it can be http://localhost:4568.  
+
+2) Through AWS aws key and secret
+- `AWS_ACCESS_KEY`: aws access key of your account.
```
Comment (therahulbhati) [inline aws-kinesis-http-connector/README.md:22]: To keep consistency with KEDA scalar object can you rename to this AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY, in fission mqt we would be creating scaledobject using KEDA convention

**Review by therahulbhati — COMMENTED** [review]
Some minor comments else LGTM.

File: aws-kinesis-http-connector/README.md (hunk 1 above)
Comment (girishg4t) [reply aws-kinesis-http-connector/README.md]: Changed

File: aws-kinesis-http-connector/go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module github.com/fission/keda-connectors/aws-kinesis-http-connector
+
+go 1.13
```
[the remark this answers is not in the document]
Comment (girishg4t) [reply aws-kinesis-http-connector/go.mod:3]: changed

File: aws-kinesis-http-connector/go.sum [hunk 3]
```diff
@@ -0,0 +1,70 @@
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```
[the remark this answers is not in the document]
Comment (girishg4t) [reply aws-kinesis-http-connector/go.sum:1]: ran the command, same changes i did for SQS

**Comment by vishal-biyani** [comment]
Fixes https://github.com/fission/keda-connectors/issues/14

**Review by therahulbhati — APPROVED** [review]
LGTM

**Comment by vishal-biyani** [comment]
One last thing @girishg4t - Can you please add it also to https://github.com/fission/keda-connectors/blob/master/README.md#current-connector-list

**Pull request merged**